### ISSUE TITLE
NPM Package missing

### ISSUE THREAD

**Issue by Dreen** [nightskylark/intl-datetime-parser#1]
This package appears to be missing from npm

https://www.npmjs.com/search?q=intl-datetime-parser

**Comment by nightskylark** [comment]
Yes, the project has not released yet. Actually, it's frozen for a while. I appreciate your interest in it. I think it will faster my work :)
